app/assets/stylesheets/components/_index-table.scss

```diff
@@ -9,6 +9,7 @@ It expects to be the primary content of the given page (full width).
 .index-table {
   // inherits from honeycrisp table until we have more design on index views
   @extend .data-table;
+  font-size: 1.6rem;
```

Comment:
The table had a weird variety of font sizes going on (one td was 15px and the rest were 16) , so I set it here directly